src/index.js

```diff
@@ -52,8 +128,10 @@ document.addEventListener('DOMContentLoaded', () => {
     bodyContent.classList.toggle('text-white');
   };
 
-  addItemButton.addEventListener('click', addItem);
-  subtractItemButton.addEventListener('click', subtractItem);
-  toggleModalButton.addEventListener('click', toggleModal);
   toggleThemeButton.addEventListener('click', toggleTheme);
-});
+};
+
+document.addEventListener('DOMContentLoaded', initModal);
```

Comment:
could we create a single function which calls all of these different functions? Or is there a reason why that would be a bad idea?

Reply:
We can. Originally I had everything happening within a single listener setup, and I pulled it apart to create a sort of separation of concerns. I don't think it is necessary here, but you've given me an idea. Stand by for incoming update!  :)

Reply:
I think that's an interesting and good approach! I was thinking we could just call each of those functions in one function. But I like your approach!